### PR TITLE
Adds selected tile counter

### LINKED ISSUE #31: Opening a new DME while there are tiles selected causes exception
Generic NullPointerException on some random un-connected line in loop(), so I'm not attaching a pic.

Not caused by my recent changes in my PR, this was ran on a clean jar.

### PR DESCRIPTION
**Pull request by d34d10cc**
Just that.

Fixes #31.

Pics:

screenshot_21

screenshot_22

screenshot_23

### CODE REVIEW

**Comment by monster860** [comment]
Maybe you should give the placement mode/handler the ability to add to the status string instead?

File: src/main/java/com/github/monster860/fastdmm/editing/SelectPlacementMode.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 public class SelectPlacementMode implements PlacementMode {
 	public FastDMM editor;
 	
-	public Set<Location> selection = new HashSet<Location>();
+	public static Set<Location> selection = new HashSet<Location>();
```
Comment (ghost) [inline src/main/java/com/github/monster860/fastdmm/editing/SelectPlacementMode.java]: no. no. no. no. no. Please don't do this, if you really need the contents of this elsewhere grab the `SelectPlacementMode` somewhere and access it trough it's reference, or put this elsewhere.

Comment (d34d10cc) [reply src/main/java/com/github/monster860/fastdmm/editing/SelectPlacementMode.java]: Yeah, it's pretty ugly. I'll do that.

**Comment by d34d10cc** [comment]
@monster860 What do you mean?

**Comment by d34d10cc** [comment]
Okay, my latest commit also fixes #31. Hooray!

**Pull request merged**